src/OpenTelemetry.Abstractions/Trace/Export/LinkList.cs

```diff
@@ -22,7 +22,7 @@ namespace OpenTelemetry.Trace.Export
 
     public sealed class LinkList : ILinks
     {
-        private static readonly LinkList Empty = new LinkList(new ILink[0], 0);
+        private static readonly LinkList Empty = new LinkList(Enumerable.Empty<ILink>(), 0);
```

Comment:
I think this should be `IEnumerable`?

Reply:
Use LinkList.Empty: https://github.com/open-telemetry/opentelemetry-dotnet/blob/4952219a8eb443582c0985021135c1121d92ff83/src/OpenTelemetry.Abstractions/Trace/Export/LinkList.cs

Reply:
@lmolkova Since this is the definition of the `Empty` field I guess here we have to stick with `Enumerable.Empty<T>`. I'm just avoiding an allocation.

Reply:
there will be ~3 allocations per process lifetime for empty (for attributes, links and events). I can tolerate 3 static allocations at startup. We can also optimize `LinksList.Empty = Enumerable.Empty<ITimedEvent<IEvent>>()` if it does not involve allocation rather than do it on every caller.

Reply:
@lmolkova I'm sorry I might be missing something but it seems the property you directed me to use is the actual property I'm changing. I don't follow, how should be using it here?

My goal was to replace the array creation with `Enumerable,Empty`. If you think it isn't worth the change, I'll drop it off the PR.

Reply:
@bruno-garcia oh, I see. YEah, please change this property.

Reply:
Thanks @lmolkova